e2e/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mcr.microsoft.com/playwright/python:v1.31.0-focal
+COPY entrypoint.sh .
+RUN apt update -y \
+    && apt install -y netcat wget \
+    && apt-get -qq -y autoremove \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN pip install pytest-playwright
+RUN curl -O https://raw.githubusercontent.com/eficode/wait-for/v2.2.3/wait-for && chmod +x ./wait-for
```

Comment:
It’s not an issue in this case, because the e2e workflow doesn’t have access to any secrets, so maybe this is a bit nitpicky, but executing bash scripts directly from the web always makes me a bit uneasy. Alternatively, we could just inline the bash script in the repo (it’s probably not going to be updated frequently)?